src/App.js

```diff
@@ -23,8 +23,8 @@ const App = () => {
   const [items, setItems] = useState([]);
   const [searchQuery, setSearchQuery] = useState("");
   const filteredPosts = filterPosts(items, searchQuery);
-  const [show, setShow] = useState(false)
-  const [deleteUser, setDeleteUser] = useState("")
+  const [show, setShow] = useState(false);
+  const [deleteUser, setDeleteUser] = useState(0);
```

Comment:
Type inconsistency - using `number` here but the variable is actually of type `user`
Will cover this when we go over TypeScript :)

Reply:
Do i just set it to empty or null then?

Reply:
You can leave it as empty

Reply:
Once we go over TypeScript, you will learn the better convention for this